jsonnet/kube-prometheus/kube-prometheus-kubeadm.libsonnet

```diff
@@ -19,14 +19,14 @@ local service(name, namespace, labels, selector, ports) = {
       'kube-controller-manager-prometheus-discovery',
       'kube-system',
       { 'k8s-app': 'kube-controller-manager' },
-      { 'component': 'kube-controller-manager' },
+      { component: 'kube-controller-manager' },
```

Comment:
nit: keys should be either in quotes or without them, mixing both looks a bit off.

Reply:
This can be fixed later though.

Reply:
This is autoformatted by `jsonnetfmt`.